versions/schematic-2.md

```diff
@@ -0,0 +1,235 @@
+**This specification is under development**
+# Sponge Schematic Specification
+
+**Version 2**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
+
+## Introduction
+
+This specification defines a format which describes an area of a [Minecraft](http://minecraft.net) world for the purpose of serialization to disk and deserialization from disk. It is designed in order to allow maximum cross-compatibility between platforms, versions, and various states of modification.
+
+## Revision History
+
+Version | Date | Notes
+---|:---:|---
+1 | 2016-08-23 | Initial Version
+2 | 2018-08-19 | More features, easier handling, see [Changelog](#changelog)
+
+## Specification
+
+### Format
+
+All field names in the specification are **case sensitive**.
+
+#### <a name="formFile"></a>File
```

Comment:
The chapters "Format" and "File" are part of the chapter "Specification".
The chapter "Specification" is split up in 2 main sections "Format" and "Schema".
So i don't really understand what you want to say.
Do you think there's something missing within the format specification (chapter "Format") if so, what exactly?
Or do you think the chapter "Format" needs to be linked within the chapter "Schema"? If so, i don't think so, because both are part of the chapter "Specification" and any implementation must respect all chapters of the "Specification".